migrations/20220412192715-create-user-verification.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('user_verifications', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      email_confirmation: {
```

Comment:
looks good but to avoid a maybe potential problem in the users model the email/username should be unique so that we dont have the same email for multiple users

Reply:
agree, will fix that in another PR as the user model is not created in this one